builder/issue.go

```diff
@@ -124,24 +141,46 @@ func (b *Issues) Tags() (tags []string) {
 
 //
 // Facts builds facts.
-func (b *Issues) Facts() (facts []api.Fact) {
-	input, err := b.read()
-	if err != nil {
+func (b *Issues) Facts() (facts api.FactMap) {
+	return
+}
+
+//
+// RuleError reported by the analyzer.
+type RuleError struct {
+	hub.SoftError
+	items []string
+}
+
+func (e *RuleError) Error() (s string) {
+	s = fmt.Sprintf(
+		"Analyser reported %d errors.",
+		len(e.items))
+	return
+}
+
+func (e *RuleError) Is(err error) (matched bool) {
+	_, matched = err.(*RuleError)
+	return
+}
+
+func (e *RuleError) Append(ruleset output.RuleSet) {
+	for s, _ := range ruleset.Errors {
+		e.items = append(e.items, s)
+	}
+}
+
+func (e *RuleError) NotEmpty() (b bool) {
+	return len(e.items) > 0
+}
+
+func (e *RuleError) Report() {
+	if len(e.items) > 0 {
```

Comment:
Looks like this should be `if len(e.items) == 0`

Reply:
Fixed.